README.md

```diff
@@ -9,6 +9,13 @@ Cerberus watches the Kubernetes/OpenShift clusters for dead nodes, system compon
 ### Install the dependencies
 ```
 $ pip3 install -r requirements.txt
+
+Cerberus uses redis to cache data in order to avoid overloading API server as well reduce the time it takes to run the checks each iteration. The redis server needs to be installed depending on the distribution and the details like host and port needs to be specified in the config as described in the usage section. Here are the commands to install redis on a Centos/Fedora/RHEL distribution:
+
+$ yum install -y redis
```

Comment:
Are we able to add the mac install versions here as well? 

`brew install redis` and `launchctl start redis`

Looking at the documentation I found below do we also need to run "redis-server" to start the redis server?
https://medium.com/@petehouston/install-and-config-redis-on-mac-os-x-via-homebrew-eb8df9a4f298